DeliBuddy/app/src/main/java/yapp/android1/delibuddy/ui/SplashActivity.kt

```diff
@@ -12,19 +12,19 @@ import yapp.android1.delibuddy.databinding.ActivitySplashBinding
 class SplashActivity : AppCompatActivity() {
     private lateinit var binding: ActivitySplashBinding
 
-    internal var handler = Handler()
+    private var handler = Handler()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivitySplashBinding.inflate(layoutInflater)
         setContentView(R.layout.activity_splash)
 
+        // TODO: 서버 개발이 끝나면 이곳에서 데이터 수신 후 intent 예정
+
         handler.postDelayed({
```

Comment:
```suggestion
        Handler().postDelayed({
```
사용처가 한 곳 밖에 없다면 변수선언이 무의미해 보입니다 !

Reply:
미처 생각하지 못한 부분이네요..! 수정하는 게 좋을 것 같아요!